<!--  -->
<launch>
    <node name="pcd_publisher" pkg="pcd_publisher" type="pcd_publisher" output="screen">
        <param name="path_to_file" value="/tmp/1484664439774713.pcd"/>
        <param name="output_topic" value="/velodyne_points_filtered"/>
        <param name="output_frame_id" value="velodyne"/>
        <param name="publishing_rate" value="10"/>

    </node>
</launch>
